jdbc-adapter/virtualschema-jdbc-adapter/src/test/java/com/exasol/adapter/jdbc/BaseRemoteMetadataReaderTest.java

```diff
@@ -250,4 +250,15 @@ void testGetSupportedTableTypes() {
         assertThat(new BaseRemoteMetadataReader(null, AdapterProperties.emptyProperties()).getSupportedTableTypes(),
                 containsInAnyOrder("TABLE", "VIEW", "SYSTEM TABLE"));
     }
+
+    @Test
+    void testConvertToOptional() {
```

Comment:
Use `@ParameterizedTest` with a `@CsvSource` since those are independent test cases. You can then also easily add a NULL test.

Reply:
I will add the NULL test. However `@CsvSource` does not work out of the box becasue I have complex Types as input (List) and output (Optional).